test/data/data-shapes/core/node/xone-duplicate.ttl

```diff
@@ -19,16 +19,16 @@
               sht:shapesGraph <xone-duplicate-shapes.ttl> ] ;
   mf:result [ rdf:type sh:ValidationReport ;
               sh:conforms "false"^^xsd:boolean ;
-	      sh:result [ rdf:type sh:ValidationResult ;
- 	      		  sh:resultSeverity sh:Violation ;
- 			  sh:focusNode ex:i ;
- 			  sh:value ex:i ;
- 			  sh:sourceShape ex:s1 ;
- 			  sh:sourceConstraintComponent sh:XoneConstraintComponent ] ;
 	      sh:result [ rdf:type sh:ValidationResult ;
  	      		  sh:resultSeverity sh:Violation ;
  			  sh:focusNode ex:j ;
  			  sh:value ex:j ;
  			  sh:sourceShape ex:s1 ;
  			  sh:sourceConstraintComponent sh:XoneConstraintComponent ] ] ;
+	      sh:result [ rdf:type sh:ValidationResult ;
+ 	      		  sh:resultSeverity sh:Violation ;
+ 			  sh:focusNode ex:i ;
+ 			  sh:value ex:i ;
+ 			  sh:sourceShape ex:s1 ;
+ 			  sh:sourceConstraintComponent sh:XoneConstraintComponent ] ;
```

Comment:
Ordering here should have no impact because the comparison is done between canonical forms of both graphs

Reply:
Still the test passes for me after reordering. Not sure why.

Reply:
Oh, I was wrong. Those results are two different blank nodes for `ex:i` and `ex:j`. Still, I thought the canonical would be should be the same...

Reply:
Aha, and those are not actually wrongly ordered results. The formatting makes it unreadable but one is nested under `mf:result` while the other is not.

Please revert this change and the rest of 8eb0d06033e0fd6c412b3441a7118e9fcba35105